plugins/eventmonitor/CMakeLists.txt

```diff
@@ -0,0 +1,35 @@
+# probe part
+if (NOT GAMMARAY_CLIENT_ONLY_BUILD)
+set(gammaray_eventmonitor_plugin_srcs
+  eventmonitor.cpp
+  eventmodel.cpp
+)
+
+gammaray_add_plugin(gammaray_eventmonitor_plugin
+  DESKTOP gammaray_eventmonitor.desktop.in
```

Comment:
anything referring to desktop files is no longer needed, this was for the old Qt4 plugin loader